certbot/src/certbot/_internal/plugins/manual.py

```diff
@@ -139,7 +139,6 @@ def has_chall(cls: Type[challenges.Challenge]) -> bool:
         resource_names = {
             challenges.DNS01: 'DNS TXT records',
             challenges.HTTP01: 'challenge files',
-            challenges.TLSALPN01: 'TLS-ALPN certificates'
```

Comment:
looks like there's also a reference to the challenge in `certbot/docs/contributing.rst`

Reply:
oop thanks!